src/extensions/resolveSource.ts

```diff
@@ -23,15 +22,20 @@ export interface UpdateWarning {
 
 /**
  * Gets the sourceUrl for a given extension name and version from the official extensions registry
+ * @param version the version of the extension
  * @param name the name of the extension.
  * @returns the source corresponding to extensionName in the registry.
  */
-export function resolveSourceUrl(registryEntry: RegistryEntry, version: string): string {
+export function resolveSourceUrl(
+  registryEntry: RegistryEntry,
+  name: string,
+  version: string
+): string {
```

Comment:
I think there's a few things here to clean up:

The documentation for this function is out of date. IIRC, the idea is that given a RegistryEntry (which is found by name and then provided to this function), get the specific source URL for a given version. Adding `name` as a parameter is unnecessary because if a RegistryEntry is provided, the name should already be present.

I think this signature change can be reverted, though the documentation can (and should) be fixed.

Reply:
The name is not in the registry entry. See https://extensions-registry.firebaseapp.com/mods.json. 

For example, a registry entry is: 
```
{
      "icons": {
        "default": "//www.gstatic.com/mobilesdk/190912_mobilesdk/bigquery_72@2x.png",
        "large": "//www.gstatic.com/mobilesdk/190912_mobilesdk/bigquery_120@2x.png",
        "small": "//www.gstatic.com/mobilesdk/190912_mobilesdk/bigquery_52@2x.png"
      },
      "labels": {
        "latest": "0.1.1"
      },
      "versions": {
        "0.1.0": "projects/firebasemods/sources/f91e8ebe-1b92-491f-ae7c-669fa9b52569",
        "0.1.1": "projects/firebasemods/sources/56b0d4b7-224a-4d8d-9472-913727baaab0"
      }
    }
```

Reply:
I _think_ that bkendall@ is referring to internal documentation about this function.  But pls let me know if there's something y'all would like me to look at.  Thx!